### PR TITLE
feat: wasmbindings for TotalBurnedNeutronsAmount #NTRN-300

### PR DESCRIPTION
**Pull request by foxpy**
Merge with https://github.com/neutron-org/neutron/pull/117 and https://github.com/neutron-org/neutron-integration-tests/pull/57

### CODE REVIEW

File: packages/neutron-sdk/src/query/total_burned_neutrons.rs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+use crate::{bindings::query::InterchainQueries, NeutronResult};
+use cosmwasm_std::{Coin, Deps};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct TotalBurnedNeutronsAmountResponse {
+    pub coins: Vec<Coin>,
```
Comment (pr0n00gler) [inline packages/neutron-sdk/src/query/total_burned_neutrons.rs:9]: Why is it a `Vec<>`?

Comment (foxpy) [reply packages/neutron-sdk/src/query/total_burned_neutrons.rs:9]: I was not sure how to handle the potential occassion of neutron denom changing, hence I decided to use a vec here.
If neutron denom shall never change, then I can remove this parameter from governance, and store here either sdk.Coin or maybe some kind of integer (which one though? there are uint64, big,Int, sdk.Int).

Comment (pr0n00gler) [reply packages/neutron-sdk/src/query/total_burned_neutrons.rs:9]: Neutron denom shouldn't be changed, so just a `Coin` is ok

**Review by albertandrejev — APPROVED** [review]
lgtm

File: packages/neutron-sdk/src/query/total_burned_neutrons.rs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+/// Returns total amount of burned neutron fees
+pub fn query_total_burned_neutrons(
+    deps: Deps<InterchainQueries>,
+) -> NeutronResult<TotalBurnedNeutronsAmountResponse> {
+    let query = InterchainQueries::TotalBurnedNeutronsAmount {};
+
+    let res: TotalBurnedNeutronsAmountResponse = deps.querier.query(&query.into())?;
+    Ok(res)
```
Comment (NeverHappened) [inline packages/neutron-sdk/src/query/total_burned_neutrons.rs:19]: maybe simplify to just `deps.querier.query(&query.into())` ?

**Review by NeverHappened — APPROVED** [review]
lgtm, 1 small comment

**Review by pr0n00gler — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Review by NeverHappened — APPROVED** [review]

**Pull request merged**